examples/combined-hbs-jsx/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  preset: 'jest-puppeteer'
+}
```

Comment:
This is repeated in the package manifest

Reply:
Fixed in this [commit](https://github.com/Financial-Times/dotcom-page-kit/pull/714/commits/10e71c2eb0acdaf6ed0163f64ce3b4d237554909).